### ISSUE TITLE
Enabling multiply features

### ISSUE THREAD

**Issue by afilippov-re** [keycloak/keycloak#16999]
### Before reporting an issue

- [X] I have searched existing issues
- [X] I have reproduced the issue with the latest release

### Area

docs

### Describe the bug

When i tried to start-dev with two features like --features=admin-fine-grained-authz,token-exchange (as in the [documentation](https://www.keycloak.org/server/features) example), I had an error:
"Option '--features' expects a single value (feature) Expected values are: authorization, account2, account-api, admin-fine-grained-authz, admin-api, admin, admin2, docker, impersonation, openshift-integration, scripts, token-exchange, web-authn, client-policies, ciba, map-storage, par, declarative-user-profile, dynamic-scopes, client-secret-rotation, step-up-authentication, recovery-codes, update-email, js-adapter, preview
Try 'kc.bat build --help' for more information on the available options."

### Version

20.0.3

### Expected behavior

Everything should work as in the example in the [documentation](https://www.keycloak.org/server/features):
you can put two arguments like "--features=docker,token-exchange" or the example needs to be changed and supplemented.

### Actual behavior

You need to split this into two commands like  --features=admin-fine-grained-authz --features=admin-fine-grained-authztoken-exchange, example isn't correct

### How to Reproduce?

Start Keycloak with  start-dev --features=admin-fine-grained-authz,token-exchange

### Anything else?

Need to double-check the behavior and change the example in the documentation if the problem is confirmed.

**Comment by sschu** [comment]
Are you running under Linux or Windows? Windows also uses comma for parameter separation...

**Comment by afilippov-re** [comment]
Yes, I running on windows and used a comma to separate, --features=admin-fine-grained-authz,token-exchange

**Comment by sschu** [comment]
This is similar to https://github.com/keycloak/keycloak/issues/16188. You can add quotes around your parameters. I provided PRs to fix that in the documentation.

**Comment by afilippov-re** [comment]
Thanks!

**Comment by jsorah** [comment]
Docs addressed as part of PR https://github.com/keycloak/keycloak/pull/16298
